PhotoScraper/RPI_SIS_PhotoScraper.py

```diff
@@ -46,98 +72,70 @@
 # Login to SIS
 def login():
     chrome_options = Options()
-    # read credentials from (optional) file
-    if len(args.credentials_file)>0 and os.path.isfile(args.credentials_file):
-        with open(str(args.credentials_file),'r') as f:
-            rin_id = f.readline().strip()
-            pin_id = f.readline().strip()
-    else:
-        rin_id = input("RIN: ")
-        pin_id = getpass.getpass("PIN: ")
 
     # By default we launch the display and allow visual debugging
     if args.headless:
         chrome_options.add_argument("--headless")
         chrome_options.add_argument("--no-sandbox")
 
-    # Just setting the default ciphers (for this session) to be weak DES/SHA for SIS compatibility
-    # Be careful about navigating to any other sites...
-    requests.packages.urllib3.util.ssl_.DEFAULT_CIPHERS = 'DES-CBC3-SHA:AES128-SHA:'+requests.packages.urllib3.util.ssl_.DEFAULT_CIPHERS
+    # read credentials from (optional) file
+    if len(args.credentials_file) > 0 and os.path.isfile(args.credentials_file):
+        with open(str(args.credentials_file), "r") as f:
+            rcsid = f.readline().strip()
+            passwd = f.readline().strip()
+    else:
+        rcsid = input("RCS ID: ")
+        passwd = getpass.getpass("Password: ")
```

Comment:
When asking for password for the website, we should make it more explicit like "SIS password" because i was confused initially if it was asking for my device password to override some permission stuff. 
```suggestion
        passwd = getpass.getpass("SIS Password: ")
```

Reply:
RCS password (it's for more than just SIS)

Reply:
Done